src/System.IO.Compression/tests/System.IO.Compression.Tests.csproj

```diff
@@ -13,6 +12,9 @@
     <Compile Include="ZipArchive\zip_UpdateTests.cs" />
     <Compile Include="Utilities\StripHeaderAndFooter.cs" />
     <Compile Include="Utilities\WrappedStream.cs" />
+    <Compile Include="ZipArchive\zip_netcoreappTests.cs" />
+    <Compile Include="CompressionStreamUnitTests.Deflate.cs" />
```

Comment:
nit: this and the following include should be at the top of the list. We usually sort the alphabetically.